stl/src/xdtest.cpp

```diff
@@ -8,12 +8,12 @@
 _EXTERN_C_UNLESS_PURE
 
 _CRTIMP2_PURE short __CLRCALL_PURE_OR_CDECL _Dtest(double* px) { // categorize *px
-    _Dval* ps = (_Dval*) (char*) px;
+    _Dval* ps = reinterpret_cast<_Dval*>(reinterpret_cast<char*>(px));

```

Comment:
Please grep for `reinterpret_cast<_[DF]val\*>\(reinterpret_cast<char\*>\(` and burn with 🔥.